esbuild.config.mjs

```diff
@@ -4,13 +4,14 @@ import * as esbuild from 'esbuild'
 const isDev = process.env.NODE_ENV !== 'DEVELOPMENT';
 
 const buildOptions = {
-    entryPoints: ['./lib/mapp.mjs', './lib/ui.mjs'],
+    entryPoints: isDev ? ['./lib/mapp.mjs', './lib/ui.mjs'] : ['./lib/mapp.mjs', './lib/ui.mjs', './tests/_mapp.test.mjs'],
```

Comment:
We bundle the test assets if we are in DEVELOPMENT mode. We can discuss if we want this all the time or in other certain situations.